cli.py

```diff
@@ -48,8 +49,27 @@ def attr_upd():
         return
 
     if args.list:
-        print(appliance_manager.aircons)
-        print(appliance_manager.washer_dryers)
+        for index, cooler in enumerate(appliance_manager.aircons):
+            print(index, cooler)
+        for index, hole in enumerate(appliance_manager.washer_dryers):
+            print(index, hole)
+        applianceelementinput = input("Enter 0-" + str(index) + " for appliance details, otherwise type 'n': ")
+        try:
+            applianceelement = int(applianceelementinput)
+        except ValueError:
+            return
+        print("Appliance selected: " + str(applianceelement))
+
+        if hasattr(appliance_manager, "aircons"):
```

Comment:
someone could have both an aircon an dryer. 
but we can improve this in the future, so I'm fine with keeping this as it is for now :)